.taskcluster.yml

```diff
@@ -36,7 +36,7 @@ tasks:
           && git config advice.detachedHead false
           && git checkout {{event.head.sha}}
           && echo "--" > .adjust_token
-          && ./gradlew clean assemble lint checkstyle ktlint pmd test
+          && ./gradlew clean assemble lint checkstyle ktlint pmd test detektCheck
```

Comment:
Is this using the config file?

Reply:
yes, it is. detekt is configured to use the config file in gradle